tests/BrowscapTest/Data/DataCollectionTest.php

```diff
@@ -26,21 +26,6 @@ public function setUp() : void
         $this->object = new DataCollection($logger);
     }
 
-    private function getPlatformsJsonFixture()
```

Comment:
I assume these are unused now (I think I recall you removing these fixture files a while back).

Reply:
yes, they were unused.